rules/windows/sysmon/sysmon_susp_office_dotnet_assembly_dll_load.yml

```diff
@@ -0,0 +1,28 @@
+title: dotNET DLL Loaded Via Office Applications
+id: ff0f2b05-09db-4095-b96d-1b75ca24894a
+status: experimental
+description: Detects any assembly DLL being loaded by an Office Product
+references:
+    - https://medium.com/threatpunter/detecting-adversary-tradecraft-with-image-load-event-logging-and-eql-8de93338c16
+author: Antonlovesdnb
+date: 2020/02/19
+tags:
+    - attack.initial_access
+    - attack.t1193
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 7
+        Image:
+            - '*\winword.exe'
+            - '*\powerpnt.exe'
+            - '*\excel.exe'
+            - '*\outlook.exe'
+        ImageLoaded:
+            - '*C:\Windows\assembly\'

```

Comment:
This looks wrong. Why isn't the `*` at the end of the expression?

Reply:
Apologies should have checked the translations more closely, I think I fixed up all the wildcard issues.